packages/query-bar/src/components/QueryUrl.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState, useRef } from 'react';
+import clsx from 'clsx';
+import {
+  Button,
+  IconButton,
+  Tooltip,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  withStyles,
+} from '@material-ui/core';
+import CopyIcon from '../assets/CopyIcon.svg';
+
+const QueryUrl = ({
+  classes,
+  filterItems,
+  localFind = {},
+  rootPath,
+}) => {
+  const [display, setDisplay] = useState(false);
+  const toggleDisplay = () => setDisplay(!display);
```

Comment:
Setting the state like this can cause issues, such as when it is rapidly toggled. This should be changed to consider the previous display.

ex.
`const toggleDisplay = () => setDisplay((prevDisplay) => !prevDisplay);`